nengo/tests/test_cache.py

```diff
@@ -425,3 +426,58 @@ def test_compare_cache_benchmark(self, varying_param, analytics_data, plt):
         plt.xlabel("Number of %s" % axis_label)
         plt.ylabel("Difference in build time (s)")
         plt.legend(loc='best')
+
+
+class TestCacheShrinkBenchmark(object):
+    n_trials = 50
+
+    setup = '''
+import numpy as np
+import nengo
+import nengo.cache
+from nengo.rc import rc
+
+model = nengo.Network(seed=1)
+with model:
+    a = nengo.networks.EnsembleArray(10, 128, 1)
+    b = nengo.networks.EnsembleArray(10, 128, 1)
+    conn = nengo.Connection(a.output, b.input)
+sim = nengo.Simulator(model)
+rc.set("decoder_cache", "size", "0KB")
+cache = nengo.cache.DecoderCache()
+    '''
+
+    stmt = 'cache.shrink()'
+
+    @pytest.mark.slow
+    @pytest.mark.noassertions
+    def test_cache_shrink_benchmark(self, analytics, logger):
+        times = timeit.repeat(
+            stmt=self.stmt, setup=self.setup, number=1, repeat=self.n_trials)
+        logger.info("Shrink took a minimum of %f seconds.", np.min(times))
+        logger.info("Shrink took a %f seconds on average.", np.mean(times))
+        logger.info(
+            "Shrink took a %f seconds on average with outliers rejected.",
+            np.mean(reject_outliers(times)))
+        analytics.add_data('times', times)
```

Comment:
Should not be required [because of `pytest_configure`](https://github.com/nengo/nengo/blob/master/nengo/conftest.py#L21). But I wonder whether the decoder cache needs to be enabled first ...?